src/components/AssetMetadata.vue

```diff
@@ -45,7 +45,7 @@ const isFavorite = computed(() => props.meta.isFavorite)
     <p>
       <span :class="{ best: best }"> {{ fileSize }} KB</span> from <code>{{ meta?.deviceId }}</code>
     </p>
-    <p>{{ resolution }} px</p>
+    <p>{{ resolution }} px<span v-if="meta.livePhotoVideoId">, <span class="best">live photo</span></span></p>
```

Comment:
Please change `class=best` to `highlight`. It's the same CSS/color, but highlighting is different to denoting the "best" asset.